packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -923,15 +925,20 @@ export async function webxdcStartUpCleanup() {
 }
 
 /**
- * Make sure a default size doesn't extend the size
- * of the primary display work area.
+ * Make sure bounds doesn't extend the primary display work area.
  */
-function adjustBounds(size: Size): Size {
+function adjustBounds(bounds: Bounds): Bounds {
   const { height: screenHeight, width: screenWidth } =
     screen.getPrimaryDisplay().workAreaSize
+  const width = Math.min(bounds.width, screenWidth)
+  const height = Math.min(bounds.height, screenHeight)
+  const x = bounds.x === -1 ? screenWidth / 2 - width : bounds.x
+  const y = bounds.y === -1 ? screenHeight / 2 : bounds.y
```

Comment:
This sets x and y like before, sort of. We might need some better tweaking here also on x and y. What happens if x is greater than screenWidth and y greater than screenHeight etc?

Reply:
I would ignore that edgecase for now until it happens in the wild